wm-system/src/main/kotlin/com/ajaxproject/warehouse/service/WaybillServiceImpl.kt

```diff
@@ -55,7 +55,7 @@ class WaybillServiceImpl(
             .map { it.productId as String }
             .minus(validIdList.toSet())
             .map { "Product with id $it not found" }
-            .toList()
+            .toList()   // TODO add product amount in DB change
```

Comment:
Please, do not commit comments.
```suggestion
            .toList()
```

Reply:
Serhii said that its OK to commit TODOs